tests/foreman/sys/test_webpack.py

```diff
@@ -6,7 +6,7 @@
 
 :Requirement: Installation
 
-:Team: Endeavour
```

Comment:
Since there's agreement on removing this component in PR https://github.com/SatelliteQE/robottelo/pull/18784, would it make sense to remove it here directly and close the older PR, since this one will be cherrypicked anyway?
cc @lhellebr